aguskoll/target-app
# Feature/sign in with facebook

### What's in

The user can log in with facebook.
The facebook token is sent to the backend. 
The user can log out from the app. 

### How it looks

https://drive.google.com/file/d/1ykKC3FKFptd4wrPoqckbtIP6vR_Z5l-P/view?usp=sharing

### How to test

Run this branch
click on the facebook button 
verify that you are being redirect to facebook
log in with the test user.
verify that you are redirected to the profile page.
